pdr_backend/sim/sim_state.py

```diff
@@ -26,6 +26,22 @@ def update(self, acc_est, acc_l, acc_u, f1, precision, recall):
         self.precisions.append(precision)
         self.recalls.append(recall)
 
+    def recent_metrics_names(self) -> List[str]:
+        return ["acc_est", "acc_l", "acc_u", "f1", "precision", "recall"]
+
+    def recent_metrics(self) -> List[Union[int,float]]:
+        """Return most recent classifier metrics"""
+        assert self.acc_ests, "must have >0 entries to call this"
+        return (
```

Comment:
I would be more comfortable with this being a dict or an Enum class. Otherwise it seems very easy to switch up the order and difficult to add more keys in the future, if needed

Reply:
I get your point.

But, this is just for dumping into a csv. I want to keep it light for now.